scripts/jenkins-yoctobuild-build.sh

```diff
@@ -569,6 +569,11 @@ build_and_test_client() {
                 sudo patch -p1 /usr/local/lib/python2.7/dist-packages/fabric/network.py b60247d78e9a7b541b3ed5de290fdeef2039c6df.patch || true
             fi
 
+	    # Zeus and older do not have this.
```

Comment:
Which layer actually have this? Is this something WIP?

Reply:
Yes, I forgot to mention. It's related to https://github.com/mendersoftware/meta-mender/pull/1037.

Reply:
I saw it now. My dependency queuing for PRs failed, hehehe.